timepiece/reports/views.py

```diff
@@ -2241,12 +2241,11 @@ def get_context_data(self, **kwargs):
                 turn_in_date = milestones.get(name='Turn-In')
                 required_completion_date = milestones.get(
                     name='Required Completion')
-                required_time = (required_completion_date.due_date -
-                    start_date.due_date) + datetime.timedelta(days=1)
-                spent_time =  (turn_in_date.due_date -
-                    start_date.due_date) + datetime.timedelta(days=1)
+                required_time = (required_completion_date.due_date - 
+                                 start_date.due_date)
+                spent_time = (turn_in_date.due_date - start_date.due_date)
```

Comment:
Are we sure we do not want to add a `1.0` here?  Throughput is zero if we don't...
Maybe what we do is if `turn_in_date.due_date == start_date.due_date` we look at time charged against the project or `ActivityGoals` against the project?